scripts/spack/packages/serac/package.py

```diff
@@ -82,10 +82,8 @@ class Serac(CachedCMakePackage, CudaPackage):
     depends_on('python', when="+devtools")
     depends_on('py-sphinx', when="+devtools")
 
-    # Workaround for concretizer bug
-    #depends_on("mfem+sundials", when="+sundials")
-    #depends_on("sundials~shared", when="+sundials")
-    depends_on("sundials~shared+hypre+monitoring")
+    depends_on("sundials~shared+hypre+monitoring~examples-c~examples-f77~examples-install",
+               when="+sundials")
```

Comment:
the `when` was slightly inconsistent.  we werent truly utilizing our sundials variant for this dependency.